index.html

```diff
@@ -0,0 +1,693 @@
+                    </div>
+                    <div class="deals-entry">
+                        <div class="discount mobile-hidden"><p>98% OFF</p></div>
+                        <img alt="Programming Ideas" src="assets/deals-img-4.jpg" class="deals-image">
+                        <div class="deals-text">
+                            <h5 class="deals-title">The Complete 2021 Microsoft Azure Certification Prep Bundle</h5>
+                            <p class="deals-old-price">$1839</p>
+                            <p class="deals-new-price">$39</p>
+                        </div>
+                    </div>
+                    <div class="deals-entry">
+                        <div class="discount mobile-hidden"><p>80% OFF</p></div>
+                        <img alt="IPVanish" src="assets/deals-img-5.jpg" class="deals-image">
+                        <div class="deals-text">
+                            <h5 class="deals-title">IPVanish VPN: 1-Yr Subscription</h5>
+                            <p class="deals-old-price">$143</p>
+                            <p class="deals-new-price">$29</p>
+                        </div>
+                    </div>
+                    <div class="deals-entry">
+                        <div class="discount mobile-hidden"><p>96% OFF</p></div>
+                        <img alt="Platinum disk" src="assets/deals-img-6.jpg" class="deals-image">
+                        <div class="deals-text">
+                            <h5 class="deals-title">The Ultimate Logic Pro X Music Production Bundle</h5>
+                            <p class="deals-old-price">$690</p>
+                            <p class="deals-new-price">$29</p>
+                        </div>
+                    </div>
+                    <div class="deals-entry">
+                        <div class="discount mobile-hidden"><p>97% OFF</p></div>
+                        <img alt="Azure Certification" src="assets/deals-img-7.jpg" class="deals-image">
+                        <div class="deals-text">
+                            <h5 class="deals-title">The 2021 Complete Microsoft Azure Certification Prep Bundle</h5>
+                            <p class="deals-old-price">$1194</p>
+                            <p class="deals-new-price">$34.99</p>
+                        </div>
+                    </div>
+                    <div class="deals-entry">
+                        <div class="discount mobile-hidden"><p>87% OFF</p></div>
+                        <img alt="Familio" src="assets/deals-img-8.jpg" class="deals-image">
+                        <div class="deals-text">
+                            <h5 class="deals-title"> Familio Private Backup Platform: Lifetime Subscription</h5>
+                            <p class="deals-old-price">$300</p>
+                            <p class="deals-new-price">$39.99</p>
+                        </div>
+                    </div>
+                </div>
+            </section>
```

Comment:
- [ ] All the article images and titles should be a link

Reply:
Noted sir!